packages/node_modules/brightspace-auth-keys-dynamodb-store/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "brightspace-auth-keys-redis-store",
+  "description": "DynamoDB based public key store implementation for brightspace-auth-keys",
+  "main": "src/public-key-store.js",
+  "scripts": {
+    "test": "nyc --all mocha --timeout 15000 --recursive ./test && npm run check-style",
+    "check-style": "eslint ."
+  },
+  "author": "D2L Corporation",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Brightspace/node-auth-keys.git"
+  },
+  "files": [
```

Comment:
:+1: `files` is strictly better than npmignore

Reply:
just copied it from your PR